ev3dev/brickpi.py

```diff
@@ -33,11 +33,13 @@
 OUTPUT_B = 'ttyAMA0:outB'
 OUTPUT_C = 'ttyAMA0:outC'
 OUTPUT_D = 'ttyAMA0:outD'
+OUTPUTS = (OUTPUT_A, OUTPUT_B, OUTPUT_C, OUTPUT_D)
```

Comment:
It is so later you can do a sanity check to verify that an OUTPUT_X is a valid output